apisix/plugins/traffic-split.lua

```diff
@@ -97,7 +97,7 @@ local upstreams_schema = {
     items = {
         type = "object",
         properties = {
-            upstream_id = schema_def.id_schema,    -- todo: support upstream_id method
+            upstream_id = schema_def.id_schema,
```

Comment:
`upstream_id` and `upstream` can be used at the same time.

Reply:
I mean, if they configure `upstream_id`, then the `upstream` should not exist and vice versa.

Reply:
I think these two methods can be supported at the same time, so that there is one more option to use upstream and upstream_id in combination. And the traffic processing between them does not conflict, they are various independent.

Reply:
Anyway, it's trivial.

Reply:
But we can limit this on Dashboard.